### ISSUE TITLE
Issue with inbound capacity

### ISSUE THREAD

**Issue by jheathco** [BlueWallet/LndHub#403]
I've got LndHub set up and when accessing via HTTP it shows my 2 well-funded open channels.  Sending/receiving payments works just fine via Zap or lncli.

I'm also able to create a lightning wallet using my lndhub URL in BlueWallet without issue.  However, upon trying to receive a payment via BlueWallet, I get an error informing me that there isn't a route or not enough capacity.  Is there some other step I'm missing here?

**Comment by Overtorment** [comment]
can you post a screenshot? i cant figure out where are you seeing this

**Comment by jheathco** [comment]
LndHub screenshot:
https://ibb.co/0scy2ny

Error in BlueWallet when trying to receive a payment:
https://ibb.co/p2TzGtY

**Comment by jheathco** [comment]
Correct, invoice created with my lndhub instance’s wallet, and attempting payment via a funded lightning wallet I’ve had with bluewallet’s default lndhub for some time. 

No issues with payment routes from lnd on my node to/from the BlueWallet lightning wallet.

**Comment by Overtorment** [comment]
route fees might exceed 1% of the amount value, thats why its failing.
Try bigger invoices, like $100

**Comment by jheathco** [comment]
Same issue - bizarre... do I need to do anything special to "activate" LndHub's ability to route payments?  It's obviously communicating with lnd properly if it's displaying the open channels, etc.  Could this be a redis issue?

**Comment by Overtorment** [comment]
are there any errors in lndhub's log?
no extra configuration is required.

post your bolt11, ill try to pay it to debug

**Comment by jheathco** [comment]
lnbc24380n1p3r8tacpp5cedj4qh7csq43r2628rmqq8zft75723xzyyyxuryp00cqffc0njsdqqcqzpgxqyz5vqsp5gnwptn4ds7kv7emxqcxx8vqgq58m8uq9wjfqxq4ywwq5z740l4as9qy9qsqdnx0wv9xavuw8vn7a9j29sfrlwhrncshfyjrxdwmyvrx6x6kltvy2tp2twpwvcjlj28lrv5hy3gzu4xwtxs2kpsek3ncwdwstjq0frsqutcgcn

Will check logs later today

**Comment by jheathco** [comment]
Don't see any errors of note... only this:

2022-03-18 03:16:45.841 [ERR] BTCN: Can't accept connection: unable to accept connection from [::1]:54974: EOF

I can see activity in logs for invoice creation, etc - all looks good.

**Comment by Overtorment** [comment]
for some reason our production node cant find path to your note __at all__

```
overtorment@lnd2:~/.lnd/data/chain/bitcoin/mainnet$ lncli queryroutes --dest 03d9d4f54573746fa4044bfd01a0f763c8754f6c980fc11cc8c983d8c2f40ade7f --amt 24380000
[lncli] rpc error: code = Unknown desc = unable to find a path to destination
```


looking at your node here https://1ml.com/node/03d9d4f54573746fa4044bfd01a0f763c8754f6c980fc11cc8c983d8c2f40ade7f i dont see any open public channels

**Comment by jheathco** [comment]
Strange - 1ML is still showing the onion address that was previously in use rather than the new public IP being used now.

**Comment by jheathco** [comment]
Here’s an invoice from lnd itself (the one lndhub is attached to). No issues paying lnd directly:

lnbc23960n1p3r2nk9pp5zmqwzwzr3dqacsz02klcaz58zc3950540dj7v36mrxsf62w5hevqdqqcqzpgxqrrssrzjqvxr7xwhgt9zjjj4cqphdvan2hpujrtpc6mt8925m0r6cxd3g8q57zcm5cqq2gsqqqqqqq05qqqq86qqeqsp5jafudmxfu0nyw8rlafz4d20v94pz3sflgqtqgzxk4xe9xxhzkzhq9qy9qsqx4ese4v8ytzl0zprl23lud4psfcg7aa893dk0paxpgx0vl2dhgvhdsp00nlgmuv6lkgrewlzhesqj66vvsth2z5usr985vujlacal5spgk0hul

**Comment by Overtorment** [comment]
looks like thats the root-cause. the way lndhub adds invoice - lnd doesnt add route hints for private channels.
PRs welcome.

**Comment by jheathco** [comment]
Yes, after a bit more investigation that appears to be the issue.  A bit of background, I was using Zap to manage the node remotely and Zap apparently defaults to private channels.  Because there were no public channels open at the time, LndHub was unable to route incoming payments (since no hints are added to the invoice).

I've since opened a few public channels and the problem was solved. :)
